tests_e2e/scenario_utils/agent_log_parser.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import print_function
```

Comment:
All the files under the "scenario_utils" provide infrastructure to execute the tests. They come directly from the 'dcr' POC, which in turn copied them from the DCR repo.

These modules require significant improvements, specially around error handling and diagnostic logging. These improvements will come on future PRs. Feel free to skip those in this initial review.